### PR TITLE
perf: do not trigger change detection when there is no `onEditorCreated` listener

### PR DESCRIPTION
**Pull request by arturovt**

### CODE REVIEW

File: projects/ngx-quill/src/lib/quill-view.component.ts [hunk 1]
```diff
@@ -152,9 +152,15 @@ export class QuillViewComponent implements AfterViewInit, OnChanges {
       this.valueSetter(this.quillEditor, this.content)
     }
 
-    // trigger created in a timeout to avoid changed models after checked
-    setTimeout(() => {
-      this.onEditorCreated.emit(this.quillEditor)
-    })
+    // The `requestAnimationFrame` triggers change detection. There's no sense to invoke the `requestAnimationFrame` if anyone is
+    // listening to the `onEditorCreated` event inside the template, for instance `<quill-view (onEditorCreated)="...">`.
+    if (this.onEditorCreated.observers.length > 0) {
```
Comment (KillerCodeMonkey) [inline projects/ngx-quill/src/lib/quill-view.component.ts:157]: could you just please use `!this.onEditorCreated.observers.length` so it is more unified with the other observers checks?

Thanks :)

**Review by KillerCodeMonkey — CHANGES_REQUESTED** [review]

File: projects/ngx-quill/src/lib/quill-view.component.ts (hunk 1 above)
Comment (arturovt) [reply projects/ngx-quill/src/lib/quill-view.component.ts]: Updated to `return` early (took from other places).

Comment (KillerCodeMonkey) [reply projects/ngx-quill/src/lib/quill-view.component.ts]: yes thats okay, but just to have a unified style to handle it. Thanks

**Review by KillerCodeMonkey — APPROVED** [review]

**Comment by KillerCodeMonkey** [comment]
@arturovt please fix the lint error :)

**Comment by arturovt** [comment]
> @arturovt please fix the lint error :)

Updated the code :)

**Pull request merged**